### DISCUSSION TITLE
nuxt3 nuxt.config.ts vite.build.assetsInlineLimit not work

### DISCUSSION THREAD

**Discussion by Alex-CZL** [nuxt/nuxt#24902] [Questions]
i just want to disable transfer inline assets, so i try to set vite.build.assetsInlineLimit to 0, but it not work.
i can still see "img src='data: base64xxxxxx'".

so what is best way to resolve this question?

**Comment by TheAlexLichter** [comment]
@Alex-CZL Best would be raising an issue and show the issue with a reproduction 👍🏻
